NEWS.md

```diff
@@ -1,5 +1,23 @@
+# BayesMallows (development versions)
+
+* Added stratified, systematic, and residual resampling to the sequential 
+  Monte Carlo algorithm. These distributions should in general be preferred to
+  multinomial resampling, which was the only available option until now.
+* The move step of the SMC algorithm now allows a user-defined lag for the 
+  sampling of latent ranks, specified in the "latent_sampling_lag" argument
+  to set_smc_options().
+* Prior for precision parameter alpha is now a gamma distribution. Until now
+  an exponential distribution has been assumed. Since the exponential is a special
+  case of the gamma with shape parameter equal to 1 (the default), this is not
+  a breaking change. However, it adds flexibility when it comes to specifying the prior.
+* setup_rank_data() now accepts a single vector of rankings, silently converting a to matrix with a single row.
+* Sequential Monte Carlo algorithm can now start from a sample from the prior
+  distribution, see the sample_prior() function for an example.
+* Added support for parallelism under-the-hood with oneTBB.
```

Comment:
Missing space on "oneTBB"? Might be an excuse to trigger another CodeFactor run 😆

Reply:
Unfortunately not ;-)

https://github.com/oneapi-src/oneTBB

Reply:
But the fix to #381 seems pretty easy, so I'll try to add that one to this PR and trigger a new run.